woocommerce/woocommerce
# Disable product inventory toggle when inventory management is disabled

### All Submissions:






### Changes proposed in this Pull Request:

Disables the toggle when site wide stock management is disabled.  Also adds a tooltip to the toggle when disabled.

<img width="574" alt="Screen Shot 2022-10-12 at 12 53 35 PM" src="https://user-images.githubusercontent.com/10561050/195438406-2f75ff8f-25e6-43a3-9e59-3428bb1c2494.png">


Closes #35046 .

### How to test the changes in this Pull Request:

1. Disable stock management at `wp-admin/admin.php?page=wc-settings&tab=products&section=inventory`
2. Navigate to `Products -> Add new (MVP)`
3. Check that the toggle to enable inventory management is disabled
4. Hover the toggle to see the tooltip
5. Enable site-wide stock management again
6. Make sure the toggle once again works
7. Check that hovering the toggle does not display the tooltip

### Other information:




### FOR PR REVIEWER ONLY:
